### ISSUE TITLE
[BUG] Already connected, not reconnecting. ​ 0 ​ Uncaught Error: connect ECONNREFUSED 127.0.0.1:55400 at TCPConnectWrap.afterConnect [as oncomplete] (​)

### ISSUE THREAD

**Issue by Kgonla** [Chia-Network/chia-blockchain#7363]
Hello,  just after upgrading to 1.2.0 my wallet never syncs:

Already connected, not reconnecting.
​ 0
​ Uncaught Error: connect ECONNREFUSED 127.0.0.1:55400
at TCPConnectWrap.afterConnect [as oncomplete] (​)

This is continuous error, retry after retry

http://fotos.subefotos.com/2229250e4fbf0caa06e81dcd1832da79o.jpg

**Comment by Kgonla** [comment]
More info:

DevTools failed to load SourceMap: Could not parse content for file:///C:/Users/Administrador/AppData/Local/chia-blockchain/app-1.2.0/resources/app.asar/build/renderer/js/493.js.map: Unexpected end of JSON input
DevTools failed to load SourceMap: Could not parse content for file:///C:/Users/Administrador/AppData/Local/chia-blockchain/app-1.2.0/resources/app.asar/build/renderer/js/main.js.map: Unexpected end of JSON input
main.js:1 Already connected, not reconnecting.
main.js:1 0
events.js:292 Uncaught Error: connect ECONNREFUSED 127.0.0.1:55400
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141)
exceptionWithHostPort @ internal/errors.js:563
afterConnect @ net.js:1141

**Comment by zanolera** [comment]
same here

**Comment by pictin** [comment]
Must be duplicate of [[BUG] events.js:292 Uncaught Error...](https://github.com/Chia-Network/chia-blockchain/issues/3869)

**Comment by CodeDeficient** [comment]
For those of you using a Linux based machine, Ubuntu or in my case Lubuntu 20.04, I may have an answer for you.
At first I thought it was a port forwarding issue but actually I think it's a connection issue to the debian update servers. I only ran into this after a linux system update. Here's what I did:

1. Go to Software Sources or Software updates (it was sources for me)
2. In "Other software" tab unmark all selections. 
3. In "Ubuntu" software tab, find the "Download from" dropbox and select "Main Server" option
4. Select close
5. Wait for "reload" prompt and select reload
6. Retry updating Chia Blockchain again like normal

This worked for me after several different attempts, I hope it works for you.
